sycl/include/CL/sycl/detail/kernel_desc.hpp

```diff
@@ -26,7 +26,8 @@ enum class kernel_param_kind_t {
   kind_accessor = 0,
   kind_std_layout = 1, // standard layout object parameters
   kind_sampler = 2,
-  kind_pointer = 3
+  kind_pointer = 3,
+  kind_specialization_constants_buffer = 4
```

Comment:
So quick NIT review (and not my section, so feel free to ignore!).  For the purposes of 'git-diff', it is really nice to leave a trailing comma on the last item.

The reason is apparent above.  The next time someone tries to look at why someone chose `kind_pointer`'s value, git blame is going to blame YOU instead of whoever touched it before!

By leaving a trailing comma, the next person through doesn't have to modify the last time.

Reply:
Good point, thanks! Will add comma to the last line after pre-commit finishes.

Reply:
Fixed